### PR TITLE
[office-js] [office-js-preview] (Excel) Pulling latest Excel type definitions from CDN

### PR DESCRIPTION
**Pull request by alison-mk**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [ ] Test the change in your own code. (Compile and run.)
- [ ] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: 
Release URL: https://appsforoffice.microsoft.com/lib/1.1/hosted/office.debug.js
Beta URL: https://appsforoffice.microsoft.com/lib/beta/hosted/office.debug.js
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

File: types/office-js/index.d.ts [hunk 1]
```diff
@@ -26267,12 +26267,12 @@ declare namespace Excel {
     /**
      * Notifies when a worksheet is moved within a workbook.
                 
-                If a worksheet is moved from one position within the workbook to another
-                via the Excel UI, then this API will trigger an event. Note that if
-                the position of a worksheet changes as a result of moving a different worksheet,
-                then this event won't trigger for both position changes. This event only triggers
-                for the primary worksheet move, and not any worksheet position changes that occur
-                as a result of that primary move.
+                 If a worksheet is moved from one position within the workbook to another
```
Comment (alison-mk) [inline types/office-js/index.d.ts:26270]: @vaquierm Both the prod and beta versions of office.d.ts on the CDN include new single spaces at the start of multiple lines, like this one. There are approx 30 groups of lines with these new single spaces, so maybe 80 lines total. 

I removed the new spaces from the beta version of office.d.ts (office-js-preview.index.d.ts) in this PR, but I left them in the prod version (office-js/index.d.ts) so you can take a look. Any idea why this happened? Thanks!

Comment (vaquierm) [reply types/office-js/index.d.ts:26270]: Looking into it.

File: types/office-js/index.d.ts [hunk 2]
```diff
@@ -30438,7 +30438,7 @@ declare namespace Excel {
         /**
          * Specifies if the password can be used to unlock worksheet protection.
                     This method doesn't change the worksheet protection state.
-                    If a password is inputted but no password is required to unlock worksheet protection, this method will return false.
+                    If a password is input but no password is required to unlock worksheet protection, this method will return false.
```
Comment (Rick-Kirkham) [inline types/office-js/index.d.ts:30441]: The Microsoft Style Guide says to use "entered" not "input" or "inputted".

**Review by Rick-Kirkham — APPROVED** [review]
Approving with one recommendation.

File: types/office-js/index.d.ts (hunk 2 above)
Comment (alison-mk) [reply types/office-js/index.d.ts:30441]: Thanks! I'll make a note to fix this in the source. It'll reach DefinitelyTyped in a few weeks or so.

**Pull request merged**